inactivity.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Maintainer and Repository Inactivity
+nav_order: 6
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Inactivity Policies
+
+As part of the normal lifecycle of a project codebases and maintainers come and go. This document formalizes a Hyperledger wide policy for archiving projects and moving maintainers to emeritus status when a project does not have an explicit policy of their own.
```

Comment:
Since we have a project lifecycle, we would not be archiving projects with this policy, but rather repositories.
```suggestion
As part of the normal lifecycle of a project codebases and maintainers come and go. This document formalizes a Hyperledger wide policy for archiving repositories and moving maintainers to emeritus status when a project does not have an explicit policy of their own.
```